modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java

```diff
@@ -135,15 +135,15 @@ public void onAfterUpdate(
 	public void onBeforeCreate(ObjectEntry objectEntry)
 		throws ModelListenerException {
 
-		_validateObjectEntry(objectEntry);
+		_validateObjectEntry(objectEntry, null);
 	}
 
 	@Override
 	public void onBeforeUpdate(
 			ObjectEntry originalObjectEntry, ObjectEntry objectEntry)
 		throws ModelListenerException {
 
-		_validateObjectEntry(objectEntry);
+		_validateObjectEntry(objectEntry, originalObjectEntry);
```

Comment:
@p-albuquerque can we keep the same order as the `onBeforeUpdate` method? That is, the `originalObjectEntry` before the `objectEntry`. Thanks!